pkgs/by-name/sh/shiori/package.nix

```diff
@@ -15,6 +15,11 @@ buildGoModule rec {
     sha256 = "sha256-gMIpDiA5ncZ50WZ2Y57mScTEXzeObgZxP+nkWe+a8Eo=";
   };
 
+  ldflags = [
+    "-X main.version=${version}"
```

Comment:
```suggestion
    "-X main.version=\"${version} (nixpkgs)\""
```

Reply:
`ldflags` doesn't seem to like spaces, and I haven't seen other uses cases with them. I've settled for 

```
Shiori version 1.7.1 (build nixpkgs-v1.7.1) at unknown
```